galaxy/accounts/models.py

```diff
@@ -68,7 +69,9 @@ class CustomUser(auth_models.AbstractBaseUser,
         db_index=True,
         help_text=_('Designates whether this user should be treated as '
                     'active. Unselect this instead of deleting accounts.'))
-    date_joined = models.DateTimeField(_('date joined'), default=timezone.now)
+    date_joined = models.DateTimeField(
+        _('date joined'),
+        default=lambda: timezone.now())
```

Comment:
Any reason for replacing function reference with lambda?

Reply:
**Yes.**
If you set `default=timezone.now` ref for `now` is stored in the model. Stubbing `timezone.now` will not work. And if I want to stub it, then it will be super tricky. 
On the other hand, `lambda: timezone.now()` encloses only `timezone`. In this case stubbing `timezone.now` will work perfectly.

Reply:
It seems reasonable to me. Please make sure though that this change doesn't affect migrations, otherwise an appropriate migration should be created.